tasks/gen/model.cr

```diff
@@ -38,6 +38,9 @@ class Gen::Model < LuckyCli::Task
 
   private def migrate_contents
     String.build do |string|
+      string << "# Information about Lucky migrations can be found at\n"
```

Comment:
How about "Learn about migrations at: https://luckyframework.org/guides/database/migrations"? so it is a bit more concise

Reply:
👍 done.